### PR TITLE
It’s about time to have a blog

### PR DESCRIPTION
**Pull request by v0lkan**
Introducing a blog article with an initial blog structure.

There‘s some navigation and templating work to be done, but we can do it in a follow-up commit.

### CODE REVIEW

**Comment by abhishek44sharma** [comment]
lgtm

File: docs/blog/2023-08-15-keep-your-secrets.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+* **Community Development and Collaboration**: We're investing in community 
+  development efforts, multi-cluster secret federation, and creating abstractions 
+  to make cloud integrations easier.
+
+Our roadmap is not just a plan; it’s a promise to our users and contributors that 
+we have a lot planned for the future of **VMware Secrets Manager**. 
+
+We’re excited about the journey ahead and invite you to be a part of it.
+
+> **⭐️ Star Us on GitHub ⭐️**
```
Comment (abhishek44sharma) [inline docs/blog/2023-08-15-keep-your-secrets.md:109]: nitpick: is it intentionally twice?

**Review by abhishek44sharma — APPROVED** [review]
lgtm

File: docs/blog/2023-08-15-keep-your-secrets.md (hunk 1 above)
Comment (v0lkan) [reply docs/blog/2023-08-15-keep-your-secrets.md:109]: :) was intentional.

Actually, I was unsure to put it left, or right. But the more stars the better :D .

Comment (v0lkan) [reply docs/blog/2023-08-15-keep-your-secrets.md:109]: Ah yes, it’s intentional. 

I want it to be visible above the fold, and then near the and after scrolling.

**Pull request merged**